src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(num) {
+      this.result += num;
+    },
+    subtract(num) {
+      this.result -= num;
+    },
+    multiply(num) {
+      this.result *= num;
+    },
+    divide(num) {
+      if (num !== 0) {
+        this.result /= num;
+      } else {
+        throw new Error('Error!');
```

Comment:
Instead of throwing a generic error with a message 'Error!', it would be more helpful to provide a descriptive message, such as 'Cannot divide by zero.' This would make it clearer to the user what went wrong.